operators/dream_texture.py

```diff
@@ -78,10 +78,12 @@ async def dream_texture(self, context):
         scene = context.scene
 
         def step_progress_update(self, context):
-            for region in context.area.regions:
-                if region.type == "UI":
-                    region.tag_redraw()
+            if hasattr(context.area,"regions"):
```

Comment:
Small formatting nitpick, I’ll add an autoformat precommit hook at some point:
```suggestion
            if hasattr(context.area, "regions"):
```